extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -249,8 +253,8 @@ export class AmpA4A extends AMP.BaseElement {
     /** @private {boolean} whether creative has been verified as AMP */
     this.isVerifiedAmpCreative_ = false;
 
-    /** @private @const {!../../../src/service/crypto-impl.Crypto} */
-    this.crypto_ = cryptoFor(this.win);
+    /** @private @const {!LegacySignatureVerifier} */
+    this.verifier_ = new LegacySignatureVerifier(this.win);
```

Comment:
Changed to `signatureVerifier_`. (I don't want to leave it as `crypto_` because readers would likely assume that it was an instance of `Crypto`, which it's not anymore.)